### DISCUSSION TITLE
Documentation

### DISCUSSION THREAD

**Discussion by vnetmx** [miten5/larawind#13] [Q&A]
Is there some documentation for this project or some official site? If not, how it was constructed and what customization are on the project to integrate Windmill.

**Comment by miten5** [comment]
Hi @vnetmx 

There is no official site for documentation, I am working on this project as hobby.
I installed Laravel Jetstream into fresh Laravel application and integrate Windmill Dashboard into it. No customization done in windmill Dashboard.

Thank you.

**Reply by intelligenceBabila** [reply]
thanks for the awesome work. I've been using windmill on my project with laravel. I was about to integrate jetstream. Now I'll use this and transfer my custom codes to it. Thanks
